components/SearchSandbox/components/RSWrapper.js

```diff
@@ -77,10 +80,10 @@ export default class RSWrapper extends Component {
 		});
 	}
 
-	getAvailableDataField = () => {
-		const { types } = propsMap[this.props.component].dataField;
+	getAvailableDataField = (fieldType) => {
+		const { types } = propsMap[this.props.component][fieldType];
 
-		if (this.props.id === 'search') {
+		if (this.props.id === 'search' && fieldType !== 'categoryField') {
```

Comment:
Please create a separate method for getting valid fields for `categoryField`. It would then be cleaner to use. No need of adding a `fieldType`.

We can share the same method as that of the list component.